src/scss/components/_header.scss

```diff
@@ -0,0 +1,222 @@
+body {
+  background-color: #f9fafb;
+  margin-top: 20px;
+}
+
+.profile-page .profile-header {
+  box-shadow: 0 0 10px 0 rgba(183, 192, 206, 0.2);
+  border: 1px solid #f2f4f9;
+}
+
+.profile-page .profile-header .cover {
+  position: relative;
+  border-radius: 0.25rem 0.25rem 0 0;
+}
+
+.profile-page .profile-header .cover figure {
+  margin-bottom: 0;
+}
+
+@media (max-width: 767px) {
+  .profile-page .profile-header .cover figure {
+    height: 110px;
+    overflow: hidden;
+  }
+}
+
+@media (min-width: 2400px) {
```

Comment:
You shouldn't really need so many media queries. BS can do a lot to change sizes of items based on viewport width, it's one of the main features.